cli/types/cypress.d.ts

```diff
@@ -3398,6 +3398,12 @@ declare namespace Cypress {
      * @internal
      */
     specPattern?: '**/*.cy.ts'
+
+    /**
+     * Raw SVG icon that will be displayed in the Project Setup Wizard. Used for third parties that
+     * want to render a custom icon.
+     */
+    icon?: string
```

Comment:
I know this is outside the scope of this PR but I wonder if we want a fallback icon of some sort